plugins/jdg72/pom.xml

```diff
@@ -26,21 +26,16 @@
             </property>
          </activation>
          <properties>
-            <version.infinispan>8.5.0.ER3-redhat-1</version.infinispan>
+            <version.infinispan>8.5.0.ER3-redhat-9</version.infinispan>
```

Comment:
What about switching to 8.5.0.ER4-redhat-10 right away? It's already available.

Reply:
Fixed